openshift/olm/knative-serving.catalogsource.yaml

```diff
@@ -606,21 +606,21 @@ data:
                           - name: CONSOLECLIDOWNLOAD_MANIFEST_PATH
                             value: deploy/resources/console_cli_download_kn_resources.yaml
                           - name: IMAGE_queue-proxy
-                            value: registry.svc.ci.openshift.org/openshift/knative-v0.14.1:knative-serving-queue
+                            value: registry.svc.ci.openshift.org/openshift/knative-nightly:knative-serving-queue
```

Comment:
Although no problem for this change, `knative-serving.catalogsource.yaml` is auto generated by `catalog.sh`

```
cd openshift/olm/ && ./catalog.sh
```

So I am hesitated to change it actually.

Reply:
It is bothering for you that `sed` to the file like?

https://github.com/openshift/knative-serving/blob/03c503eaaa67037b8dcadcd9816879d48e193cce/openshift/e2e-common.sh#L98-L104

Reply:
Nope, thanks for the pointer, now replacing the image refs in client CI setup.